Backpack/Chip/Classes/BPKChip.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * `BPKChip` is a subclass of `UIControl` configured with Skyscanner style properties.
+ */
+NS_SWIFT_NAME(Chip) IB_DESIGNABLE @interface BPKChip: UIControl
+
+/**
+ * The title to display inside the chip.
+ */
+@property(nonatomic, copy, nullable) NSString *title;
+
+/**
+ * Creates a `BPKChip` with a decoder (typically when creating form Storyboards)
+ *
+ * @param enabled BOOL the new value
+ */
+- (void)setEnabled:(BOOL)enabled;
```

Comment:
Why are we declaring this at all? It's inherited from `UIControl` right?

Reply:
Yeah it comes from UIControl
If we don't declare this, I don't seem to be able to set `chip.enabled` or use `chip.setEnabled`. Am I doing something wrong?

Reply:
Yeah that seems odd you should be able to still use those without it declared. I think the getter is `isEnabled` though however `chip.enabled = true` should work from Swift

Reply:
When I remove this, it's not happy with `chip.enabled = true` but `chip.isEnabled = true` works

Reply:
That sounds correct if it's fro Swift. The whole property has been renamed `isEnabled` in Swift